docs/.vuepress/theme/Layout.vue

```diff
@@ -134,12 +121,45 @@ export default {
       this.isSidebarOpen = false
     })
   },
+  // Causing errors on route change -- just updating meta tags so I think we're ok
+  // watch: {
+  //   '$page': 'updateMeta',
+  // },
 
   beforeDestroy () {
-    updateMetaTags(null, this.currentMetaTags)
+    this.updateMetaTags(null, this.currentMetaTags)
   },
 
   methods: {
+    updateMeta() {
+      document.title = this.$title
+      document.documentElement.lang = this.$lang
+      const meta = [
+        {
+          name: 'description',
+          content: this.$description
+        },
+        ...(this.$page.frontmatter.meta || [])
+      ]
+      this.currentMetaTags = this.updateMetaTags(meta, this.currentMetaTags)
+    },
+    updateMetaTags (meta, current) {
+      if (current) {
+        current.forEach(c => {
+          document.head.removeChild(c)
```

Comment:
i was just fixing that bug that gets thrown on this line all the time, i'll wait till this merges and make the change here

Reply:
I commented out the watch that was partially responsible, just FYI